dsl-interpreter/src/activities.ts

```diff
@@ -0,0 +1,28 @@
+export async function Activity1(arg1: string): Promise<string> {
```

Comment:
Is this a new convention using pascal case for activity functions?

Reply:
no just felt like copying the Go example directly which names those activities.